README.md

```diff
@@ -14,30 +28,99 @@ To install multiple versions of Node.js, you may find it easier to use a node ve
 * [nvm](https://github.com/creationix/nvm)
 * [n](https://github.com/tj/n)
 
-### Installation
+
+## Installation
+
+Clone this repo and install its dependencies...
```

Comment:
I think it makes it clear that the following code block is related to the instruction